docs/3_NEW_TEMPLATE.md

```diff
@@ -1,15 +1,406 @@
-# 3. Creating a New Template
-
-> *Have you completed the quick start guide? Deploy your first infrastructure as code project with Cobalt by following the [quick-start guide](./2_QUICK_START_GUIDE.md).*
+# 3. Cobalt Templating From Scratch
 
 ## 3.1 Overview
-< Describe the general context around this walkthrough/lab >
+
+Per the [quickstart guide](./2_QUICK_START_GUIDE.md), we demonstrated that you can get pretty far with *Cobalt* by simply relying on any out of the box *Cobalt Infrastructure Template* or *CIT* (/kɪt/). However, for several reasons, you may very well have unique infrastructure scenarios that require you to extract more use out of *Cobalt*. Therefore, we recommend building a *Cobalt Infrastructure Template* from scratch in order to cultivate a deeper understanding of what *Cobalt* has to offer.
```

Comment:
Here are some suggestions for making this a little less wordy. I like the overall intent a lot!
```suggestion
Per the [quickstart guide](./2_QUICK_START_GUIDE.md), we demonstrated how easy it is to deploy an existing *Cobalt Infrastructure Template* or *CIT* (/kɪt/). However, it is likely that you will need to develop your own *CIT* by composing the foundational infrastructure modules that ship with *Cobalt*. This guide will walk you through building a *Cobalt Infrastructure Template* from scratch in order to cultivate a deeper understanding of what *Cobalt* has to offer.
```